tests/dapr_tests.mk

```diff
@@ -309,6 +309,7 @@ setup-test-components: setup-app-configurations
 	$(KUBECTL) apply -f ./tests/config/app_topic_subscription_pubsub_grpc.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/kubernetes_allowlists_config.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/kubernetes_allowlists_grpc_config.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/dapr_redis_state_query.yaml --namespace $(DAPR_TEST_NAMESPACE)
```

Comment:
Can we use the existing DAPR_TEST_QUERY_STATE_STORE attribute to use redis instead of creating a new one?

Reply:
DAPR_TEST_QUERY_STATE_STORE is used to alternate between MongoDB and CosmosDB.
The Redis query e2e test is an additional test.

Reply:
E2E tests are not component specific.

Reply:
No problem, I will replace mongodb with redis

Reply:
We can keep a mix of both and use both in our kind tests.

Reply:
> We can keep a mix of both and use both in our kind tests.

I don't think so. We only can have one component with the name `querystatestore`, and we already alternate it between AKS (CosmosDB) and KIND (MongoDB) e2e tests. Beside, if e2e tests are component-agnostic, it should not matter which component is used.

Reply:
Running E2E tests with the same code but against different components can help uncover inconsistencies. I will look into that later. For now, let's just keep one configured component for query state. I will look into doing a combination of those later for Kind. Thanks.

Reply:
> Running E2E tests with the same code but against different components can help uncover inconsistencies. I will look into that later. For now, let's just keep one configured component for query state. I will look into doing a combination of those later for Kind. Thanks.

I actually agree that testing more components is beneficial. This is why I added Redis component to the existing ones ...

Reply:
My point is more in the how. The idea is that we keep the code component independent and configure the different components outside the test code. Similar to how we switch the state store between redis and cosmosdb today.

Reply:
I made a cosmetic change in an attempt to keep both Redis and MongoDB query tests. LMK.